### PR TITLE
correctly serialize loss function (one liner)

### PR DESCRIPTION
**Pull request by cmyr**
This fixes a bug I introduced during review of #2690, which caused the loss function to be incorrectly serialized. 😞

### CODE REVIEW

File: keras/engine/topology.py [hunk 1]
```diff
@@ -2358,7 +2358,7 @@ def _updated_config(self):
 
         if hasattr(self, 'optimizer'):
             model_config['optimizer'] = self.optimizer.get_config()
-            model_config['loss'] = self.loss.__class__.__name__
+            model_config['loss'] = getattr(self.loss, '__name__', self.loss)
```
Comment (cmyr) [inline keras/engine/topology.py]: if we were passed a function, we save its name. Else we assume we were passed a string.
